src/playground/sample_data_ingestion/config/ad_plugin/ad_plugin_create_payload.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
where is the cronjob yaml files ? please add cronjob yaml file as well to config/playground/cronjobs/{your tool /function}/your yaml file. Thanks

Reply:
Done, the yaml files are added. I may have to change the image name, depending on where the image is hosted.

Reply:
Done, both are split into `cronjob` and `job` directories respectively.